auto_round/__main__.py

```diff
@@ -309,7 +298,7 @@ def tune(args):
         low_cpu_mem_usage=low_cpu_mem_usage, data_type=args.data_type,
         enable_norm_bias_tuning=args.enable_norm_bias_tuning)
     model, _ = autoround.quantize()
-    model_name = args.model_name.rstrip("/")
+    model_name = args.model.rstrip("/")
     if args.low_cpu_mem_mode == 1 or args.low_cpu_mem_mode == 2:
```

Comment:
bug when user need to export multiple formats, as the same save folder is using,